script/github-actions/code-coverage-format-report.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const codeCoverageReport = fs.readFileSync(
   path.join(
     __dirname,
-    '../../qa-standards-dashboard-data/coverage/coverage_report.txt',
+    '../../qa-standards-dashboard-data/coverage/coverage_report.txtt',
```

Comment:
Is this extra 't' in 'txtt' intentional?

Reply:
sorry - this PR, while marked "ready for review", has to do with changes to the pull request workflow which only runs once a PR is "ready". This is an intentionally introduced change just to trigger the linter to check this file. PR not technically "ready" yet.